modules/core/src/main/scala/org/tessellation/config/types.scala

```diff
@@ -61,17 +62,22 @@ object types {
     remainingWeight: Weight
   )
 
+  case class OneTimeReward(epoch: EpochProgress, address: Address, txnAmount: TransactionAmount)
+
   case class RewardsConfig(
     programs: EpochProgress => ProgramsDistributionConfig = mainnetProgramsDistributionConfig,
-    rewardsPerEpoch: SortedMap[EpochProgress, Amount] = mainnetRewardsPerEpoch
+    rewardsPerEpoch: SortedMap[EpochProgress, Amount] = mainnetRewardsPerEpoch,
+    oneTimeRewards: List[OneTimeReward] = List.empty
   )
 
   object RewardsConfig {
-    val stardustPrimary: Address = Address("DAGSTARDUSTCOLLECTIVEHZOIPHXZUBFGNXWJETZVSPAPAHMLXS")
+    import org.tessellation.sdk.domain.transaction.TransactionValidator.stardustPrimary
+    val stardustNewPrimary = Address("DAG8vD8BUhCpTnYXEadQVGhHjgxEZZiafbzwmKKh")
```

Comment:
if you removed `stardustPrimary`, you can name `stardustNewPrimary` as just `stardustPrimary` 🤔

Reply:
The distinction here seems fine to make since it's an updated value from the original

Reply:
yeah, I like the distinction, but I don't like the direction it goes: `stardustPrimary`, `stardustNewPrimary`, ..., `stardustNewerPrimary`, `stardustFinalPrimary`, `stardustMoreFinalPrimary` etc :D